src/Tickets/Emails/Email_Abstract.php

```diff
@@ -873,4 +874,15 @@ public function get_post_type_data(): array {
 
 		return $data;
 	}
+
+	/**
+	 * Get the related JSON data for this email.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function get_json_data(): array {
+		return Handler::get_data( $this );
```

Comment:
We'll need to be able to filter this data at some point, so that we can inject the event data for example.